gulp/tasks/default.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+
+gulp.task( 'default',
+  [
+    'lint:src',
+    'test:unit',
+    'test:macro',
+    'test:processor',
```

Comment:
May not matter, they're gone as of https://github.com/cfpb/cfgov-refresh/pull/901